Sources/StreamChatUI/Resources/en.lproj/Localizable.strings

```diff
@@ -2,107 +2,215 @@
+/// Shown in the message actions popup for the edit message action.
 "message.actions.edit" = "Edit Message";
+/// Shown in the message actions popup for the copy message action.
 "message.actions.copy" = "Copy Message";
+/// Shown in the message actions popup for the delete message action.
 "message.actions.delete" = "Delete Message";
+/// Shown as a title in the confirmation popup when deleting a message.
 "message.actions.delete.confirmation-title" = "Delete Message";
+/// Shown as a text in the confirmation popup when deleting a message.
 "message.actions.delete.confirmation-message" = "Are you sure you want to permanently delete this message?";
+/// Shown in the message actions popup for the unblock user action.
 "message.actions.user-unblock" = "Unblock User";
+/// Shown in the message actions popup for the block user action.
 "message.actions.user-block" = "Block User";
+/// Shown in the message actions popup for the unmute user action.
 "message.actions.user-unmute" = "Unmute User";
+/// Shown in the message actions popup for the mute user action.
 "message.actions.user-mute" = "Mute User";
+/// Shown in the message actions popup for the resend message action.
+/// Displayed only when the message was not sent successfully.
 "message.actions.resend" = "Resend";
+/// Shown in the message actions popup for the flag message action.
 "message.actions.flag" = "Flag Message";
+/// Shown as a text in the confirmation popup when flagging a message.
 "message.actions.flag.confirmation-title" = "Flag Message";
+/// Shown as a text in the confirmation popup when flagging a message.
 "message.actions.flag.confirmation-message" = "Do you want to send a copy of this message to a moderator for further investigation?";
+/// Shown in the message actions popup for marking message as unread.
 "message.actions.mark-unread" = "Mark as unread";
 
+/// Shown as a title in the popup shown when a message was bounced due to moderation.
 "message.moderation.title" = "Are you sure?";
+/// Shown as a text in the popup shown when a message was bounced due to moderation.
 "message.moderation.message" = "Consider how your comment might make others feel and be sure to follow our Community Guidelines.";
+/// Shown as a button text in the popup shown when a message was bounced due to moderation.
 "message.moderation.resend" = "Send Anyway";
+/// Shown as an edit message action in the popup shown when a message was bounced due to moderation.
 "message.moderation.edit" = "Edit Message";
+/// Shown as a delete message action in the popup shown when a message was bounced due to moderation.
 "message.moderation.delete" = "Delete Message";
 
+/// Shown when a user is online.
 "message.title.online" = "Online";
+/// Shown when a user is offline.
 "message.title.offline" = "Offline";
+/// Shown in a group chat title.
+/// The first placeholder is the number of members, while the second one shows how many are online.
 "message.title.group" = "%d members, %d online";
```

Comment:
Ideally we should use `%1$@` and `%2$@` here for allowing to reorder arguments per locale. But I guess it is actually fine in this case.

Reply:
Yes, it's fine. Also, not touching the content in this PR, just adding comments to the texts.